api/all/src/main/java/io/opentelemetry/api/baggage/Baggage.java

```diff
@@ -99,4 +99,13 @@ default boolean isEmpty() {
    * be set to not use an implicit parent, so any parent assignment must be done manually.
    */
   BaggageBuilder toBuilder();
+
+  /**
+   * Returns the {@code BaggageEntry} associated with the given key.
+   *
+   * @param entryKey entry key to return the {@code BaggageEntry} for, or {@code null} if no {@code
+   *     Entry} with the given {@code entryKey} is in this {@code Baggage}.
+   */
+  @Nullable
+  BaggageEntry getEntry(String entryKey);
```

Comment:
oh, I think we have to provide a default here

Reply:
oh good call...because without it we break existing implementations. I had one, and then removed it. :) Will add back.